### PR TITLE
MWPW-118876 Rewrite relative links temp fix

### PR DESCRIPTION
**Pull request by Brandon32**
* Rewrite relative links for PDFs and HTML (Note: PDF SDK still has CORS issue to be resolved by WPS change)

Resolves: [MWPW-118876](https://jira.corp.adobe.com/browse/MWPW-118876)

**Test URLs:**
- Before: https://main--bacom--adobecom.hlx.page/drafts/iscandar/fake-gated-thank-you-page?martech=off
- After: https://rewrite-fix--bacom--adobecom.hlx.page/drafts/iscandar/fake-gated-thank-you-page?martech=off
- Before: https://main--bacom--adobecom.hlx.page/de/customer-success-stories/fraport-case-study?martech=off
- After: https://rewrite-fix--bacom--adobecom.hlx.page/de/customer-success-stories/fraport-case-study?martech=off

### CODE REVIEW

File: scripts/utils.js [hunk 1]
```diff
@@ -10,6 +10,27 @@
...
  */
 
+const ORIGIN = 'https://business.adobe.com';
+
+// Temporary rewrite for relative URLs back to fully qualified
+// https://github.com/adobe/helix-html-pipeline/blob/main/src/steps/utils.js
+export const rewriteUrl = (url) => {
+  if (!url) {
+    return url;
+  }
```
Comment (meganthecoder) [inline scripts/utils.js:20]: nit:
```suggestion
  if (!url) return url;
```

File: scripts/utils.js [hunk 2]
```diff
@@ -10,6 +10,27 @@
...
+
+  const { host, pathname, search, hash } = new URL(url);
+
+  // eslint-disable-next-line no-restricted-globals
+  if (host === location.host) {
+    if (pathname.endsWith('.pdf') || pathname.endsWith('.html')) {
+      return `${ORIGIN}${pathname}${search}${hash}`;
+    }
+  }
```
Comment (meganthecoder) [inline scripts/utils.js:29]:
```suggestion
  if (host === location.host && (pathname.endsWith('.pdf') || pathname.endsWith('.html')) {
    return `${ORIGIN}${pathname}${search}${hash}`;
  }
```

File: scripts/utils.js (hunk 1 above)
Comment (Brandon32) [reply scripts/utils.js:20]: 👍 Sounds good

File: scripts/utils.js (hunk 2 above)
Comment (Brandon32) [reply scripts/utils.js:29]: 👍 It's a little long but still seems easy enough to read.

**Review by meganthecoder — APPROVED** [review]

**Comment by Brandon32** [comment]
Fixed by https://github.com/adobe/helix-html-pipeline/pull/175

**Pull request closed without merging**